scripts/warn-about-yarn1.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+const {execSync} = require(`child_process`);
+
+console.log(`Your global Yarn binary isn't recent enough; please upgrade to 1.17.2 or higher.`);
+console.log(`Binary: ${execSync(`which yarn`).toString().trim()} (${execSync(`yarn --version`).toString().trim()})`);
```

Comment:
This will fail on Windows no?

The equivalent [seems](https://superuser.com/questions/207707/what-is-windows-equivalent-of-the-which-command-in-unix-is-there-an-equivale) to be `where`

Reply:
Good point - fixed 👍